### ISSUE TITLE
Weird results for different models

### ISSUE THREAD

**Issue by Annieliaquat** [tensorflow/models#10864]
I am training 3 different models for covid-19 detection SSD MobileNetV2 FPNLite 320x320,SSD MobileNetV2 FPNLite 640x640 ,SSD MobileNetV2  320x320. I have trained each model for 4000 steps, I have got good but some weird results. 

![image](https://user-images.githubusercontent.com/43025113/208229263-5f567b24-44ba-4caa-a1e5-d0c12db95adc.png)

My second models has gotten less loss value but I am confused why it has gotten less mAP values than other two models. The learning rate for all three models are the same as in the pipeline config

**Comment by laxmareddyp** [comment]
Hi @fyangf,

Could you please look into this issue.

Thanks

**Comment by fyangf** [comment]
It is possible that lower loss does not translate to higher mAP, because for object detection, the loss is minimized regarding class label and box localization. It could be possible that many boxes are perfectly localized but they get wrong class prediction. In this case, you may see lower total loss but the mAP may not be high.

Also, I think you are using Object Detection API but not the models under `official`. Note that we are maintaining `official` models but not Object Detection API. So there could also be issues in Object Detection API that is beyond our scope.
